My Spotify/My Spotify/Network Layer/APIManager.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ApiManager.swift
+//  My Spotify
+//
+//  Created by Devarsh Bhalara on 04/07/23.
+//
+
+import Foundation
+import Alamofire
+import UIKit
+import Reachability
+
+class APIManagerDemo {
+    
+    init() { }
+    
+    // MARK: Vars & Lets
+    private let sessionManager: Session = Session()
+    static var shared = APIManagerDemo()
+    let reachability = try? Reachability()
+    static let errorCodeList =  [400, 401, 402, 403, 404, 405, 406, 407, 408, 409, 410, 411, 412, 413, 414, 415, 416, 417, 418, 421, 422, 423, 424, 425, 426, 427, 428, 429, 431, 451, 500, -1009, -1001]
+    
+    //MARK: - Methods
```

Comment:
Either keep `-` or not. Keep code standard.
```suggestion
    //MARK: Methods
```